src/client.ts

```diff
@@ -13,11 +13,11 @@ import {
   RequestContext,
   ResponseContext,
   Placement,
-} from './generated';
+  RequiredError,
+  UserdbApi
+} from '@adzerk/api-decision-js';
```

Comment:
I'm happy that you removed the generated code from the repo! Much cleaner and less confusing.

Reply:
thank you